CHANGELOG.md

```diff
@@ -1,14 +1,34 @@
 ## Unreleased
 
-* [FEATURE] Add `-kubelet-endpointslice` argument to support `EndpointSlice` for the kubelet controller.
+## 0.77.0 / 2024-09-19
+
+> Along with this release we also introduce a new command line tool [poctl](https://github.com/prometheus-operator/poctl). It is designed specifically for managing Prometheus Operator Custom Resources.
```

Comment:
Nothing required, but would be nice if we callouts: https://github.com/orgs/community/discussions/16925

info and warning wdyt?

Reply:
This is nice. I had thought about updating our README as well when I saw something similar in metrics-server repo.

I will update. Thanks for the suggestion 🙂

Reply:
https://github.com/prometheus-operator/prometheus-operator/blob/0a82836229afad0c1044dfcc0d03d3c8fb13f15f/CHANGELOG.md looks nice :D